Swagger failing validation on Travis

The output is
```bash
$ make swagger-valid
node_modules/swagger-cli/bin/swagger-cli.js validate docs/swagger.yml
Validation failed. Property 'results' listed as required but does not exist in '/definitions/ListPagination'
make: *** [swagger-valid] Error 1
```

Looking at `docs/swagger.yml`, the error seems to be correct.

--- a/docs/swagger.yml
+++ b/docs/swagger.yml
@@ -1242,7 +1242,6 @@ definitions:
       - count
       - next
       - previous
-      - results
     properties:
       count:
         type: "integer"

